scripts/bootstrap.js

```diff
@@ -0,0 +1,24 @@
+const path = require('path');
```

Comment:
What is this file `bootstrap` for?

Reply:
These are some scripts taken from `react-native-builder-bob`.

With this bootstrap we can run the example app directly from root directory (no need to navigate to example directory)